.github/taskdef/prod-xlayer-mainnet.yaml

```diff
@@ -0,0 +1,34 @@
+region: eu-west-1
+hostport: 80
+containerport: 80
+role: backend-apps
+environment: prod
+iac: terraform-workspace-aws-prod-apps-eu-west-1-apps-xlayer-mainnet-auto-claim-polygon-technology
```

Comment:
Should be 
terraform-workspace-aws-prod-applications-eu-west-1-apps-auto-claim-xlayer-mainnet-auto-claim